packages/swapper/src/swappers/thorchain/utils/estimateTradeFee/estimateTradeFee.ts

```diff
@@ -87,34 +62,11 @@ export const estimateTradeFee = async (
 
   const buyFeeAsset = new AssetService().getAll()[buyFeeAssetId]
 
-  switch (buyChainId) {
-    case 'bip122:000000000019d6689c085ae165831e93':
-      return fromBaseUnit(
-        bnOrZero(btcEstimate(gasRate)).times(feeAssetRatio).dp(0),
-        buyFeeAsset.precision
-      )
-    case 'eip155:1':
-      switch (assetNamespace) {
-        case 'slip44':
-          return fromBaseUnit(
-            bnOrZero(ethEstimate(gasRate)).times(feeAssetRatio).dp(0),
-            buyFeeAsset.precision
-          )
-        case 'erc20':
-          return fromBaseUnit(
-            bnOrZero(erc20Estimate(gasRate)).times(feeAssetRatio).dp(0),
-            buyFeeAsset.precision
-          )
-        default:
-          throw new SwapError('[estimateTradeFee] - unsupported asset namespace', {
-            code: SwapErrorTypes.VALIDATION_FAILED,
-            details: { assetNamespace }
-          })
-      }
-    default:
-      throw new SwapError('[estimateTradeFee] - unsupported chain id', {
-        code: SwapErrorTypes.VALIDATION_FAILED,
-        details: { chainId: buyChainId }
-      })
-  }
+  return fromBaseUnit(
+    THOR_TRADE_FEE_MULTIPLIERS[buyChainId as SUPPORTED_BUY_CHAINS]
+      .times(buyFeeAssetRatio)
+      .times(gasRate)
+      .dp(0),
+    buyFeeAsset.precision
```

Comment:
is this always going to be `THORCHAIN_FIXED_PRECISION`?

Reply:
maybe we should for consistency now that its using the map. will need to change the hard coded eth and atom values but seems a bit cleaner

Reply:
maybe better:

dont use fromBaseUnit() and have the multiplier values account for it all